src/app/cli/src/commands/system_info_command.rs

```diff
@@ -133,7 +133,7 @@ pub struct SystemInfo {
 impl SystemInfo {
     pub async fn collect(
         container_runtime_svc: Arc<ContainerRuntime>,
-        workspace_svc: Arc<WorkspaceService>,
+        workspace_svc: &Arc<WorkspaceService>,
```

Comment:
Similarly here, might be enough to pass `&WorkspaceService`

Reply:
Done